### PR TITLE
The Clean Input Data tool failed to find en dash

### PR DESCRIPTION
**Pull request by SNoiraud**
Fixes [#11308](https://gramps-project.org/bugs/view.php?id=11308)

### CODE REVIEW

**Comment by lordemannd** [comment]
do we need to also look for the em dash?

File: gramps/plugins/tool/removespaces.py [hunk 1]
```diff
@@ -216,11 +216,35 @@ def places(self):
...
+            if plat.find('—') != -1: # em dash (u+2014)
+                found = True
+            if plat.find('―') != -1: # horizontal bar (u+2015)
+                found = True
             plon = place.get_longitude()
             if plon != plon.strip():
                 found = True
             if plon.find(',') != -1:
                 found = True
+            if plon.find('‐') != -1: # hyphen (u+2010)
```
Comment (Nick-Hall) [inline gramps/plugins/tool/removespaces.py]: It would be neater to write a function rather than to duplicate code.

**Comment by lordemannd** [comment]
i just noticed a change in the editor for the lat and long, any edit to the field always leaves the cursor at the end of the field. this issue appeared between 5.0.1 and 5.0.2. I have created a bug for it.
I might be related to the cleanup changes?

File: gramps/plugins/tool/removespaces.py [hunk 2]
```diff
@@ -53,6 +53,27 @@
 WIKI_HELP_PAGE = '%s_-_Tools' % URL_MANUAL_PAGE
 WIKI_HELP_SEC = _('manual|Remove_leading_and_trailing_spaces')
 
+def validate_lat_lon(field):
```
Comment (Nick-Hall) [inline gramps/plugins/tool/removespaces.py]: This function could further be simplified to:
```
def validate_lat_lon(field):
    for char in (',', '‐', '‑', '‒', '–', '—', '―'):
        if field.find(char) != -1:
            return True
    return False
```

Comment (SNoiraud) [reply gramps/plugins/tool/removespaces.py]: I'll do that

**Pull request merged**